### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by code-terror**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone.

File: Mayhem/fuzz_encodings.mayhemfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+project: code-terror/encoding_rs
```
Comment (xansec) [inline Mayhem/fuzz_encodings.mayhemfile:1]: Please use generic project names.

**Review by xansec — CHANGES_REQUESTED** [review]
Make sure project name for each Mayhemfile is generic. Please make the requested changes.

File: Dockerfile (hunk 1 above)
Comment (code-terror) [reply Dockerfile:1]: Need to use git clone for the second repository, or will have to modify cargo.toml file.. Git cloning is better for the second repository

**Comment by code-terror** [comment]
@xansec Done

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (xansec) [inline Dockerfile:1]: Right, I was mainly referring to the eponymous repo. This seems fine.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**